HOK.Elastic.FileSystemCrawler/WorkerArchiveDiscover.cs

```diff
@@ -0,0 +1,182 @@
+﻿using HOK.Elastic.DAL;
```

Comment:
It doesn't appear this file is being used by the solution. Was this used earlier but isn't anymore?